odata-csdl/14 Vocabulary and Annotation.md

```diff
@@ -1699,12 +1699,10 @@ types](#BuiltInTypesfordefiningVocabularyTerms)
 `Edm.NavigationPropertyPath`, `Edm.AnyPropertyPath`, or `Edm.ModelElementPath`.
 Its argument is a [model path](#PathExpressions) with the following
 restriction:
-- A non-null path MUST resolve to a model
-element whose type is an entity type, or a collection of entity types,
-e.g. a navigation property.
+- A non-null path MUST end with a [navigation property](#NavigationProperty)
+or a term cast to a term whose type is an entity type or a collection of entity types.
 
-The value of the navigation property path expression is the path itself,
-not the entitiy or collection of entities identified by the path.
+The value of the navigation property path expression is the navigation property or term cast, not its instances.
```

Comment:
```suggestion
The value of the navigation property path expression is the navigation property (not its instances) or the term (not the annotation with that term).
```

Reply:
Reverted this part of the change to the V4.01 text